app/src/main/java/swati4star/createpdf/activity/MainActivity.java

```diff
@@ -369,87 +371,109 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
         switch (item.getItemId()) {
             case R.id.nav_home:
                 fragment = new HomeFragment();
+                setTitle(R.string.app_name);
```

Comment:
Instead of having this in every switch-case, can we have a hashmap and just use that.
Also, you havent changed the title for `nav_add_watermark`

Reply:
Sorry i didn't get you

Reply:
Check out the `fillMap()` method of `HomeFragment` class. That should be enough to make Swati's comment clear to you.

Reply:
how can i use hashmap for setting title 
i didn't understand can you please share some reference for this

Reply:
@sidhuparas Help @bharatmk257

Reply:
@bharatmk257 As I wrote in one of the comments above `fillMap()` method of `HomeFragment` class is perfect example. Check that code and ask me any specific question.